doc/api/https.md

```diff
@@ -82,19 +111,45 @@ https.createServer(options, (req, res) => {
 <!-- YAML
 added: v0.1.90
 -->
+- `callback` {Function}
 
 See [`http.close()`][] for details.
 
 ### server.listen(handle[, callback])
+- `handle` {Object}
+- `callback` {Function}
+
 ### server.listen(path[, callback])
-### server.listen(port[, host][, backlog][, callback])
+- `path` {string}
+- `callback` {Function}
+
+### server.listen([port][, host][, backlog][, callback])
+- `port` {number}
+- `hostname` {string}
+- `backlog` {number}
+- `callback` {Function}
 
 See [`http.listen()`][] for details.
 
-## https.get(options, callback)
+## https.get(options[, callback])
 <!-- YAML
 added: v0.3.6
 -->
+- `options` {Object | string}
+  - `protocol` {string} Default = `https:`
+  - `host` {string} Default = `localhost`
+  - `hostname` {string} Alias for `host`.
+  - `family` {number}
+  - `port` {number} Default = 80
+  - `localAddress` {string}
+  - `socketPath` {string}
+  - `path` {string} Default = `/`
+  - `headers` {Object}
+  - `auth` {string}
+  - `agent` {Agent | boolean} Default = `globalAgent`
```

Comment:
`https.globalAgent` would be clearer(notice `https.globalAgent !== http.globalAgent`)